src/intl.ts

```diff
@@ -55,6 +55,9 @@ export const commonMessages = defineMessages({
   lastName: {
     defaultMessage: "Last Name"
   },
+  limitReached: {
+    defaultMessage: "Reached limit for this plan"
```

Comment:
It should explicitly say: "Already at the limit. Unable to create more users." No need to refer to "this plan."